### ISSUE TITLE
Custom compile doesn´t work

### ISSUE THREAD

**Issue by guillo270120** [FlutterFlow/flutterflow-issues#457]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)
The application does not want to compile the custom code, it is only 1 custom action to generate a uuid which was working and suddenly it asked me to compile and it shows errors which it does not have, I urgently need your help.

## Current behavior (required)
It doesn't compile the custom actions, it marks errors that don't already exist, I even tried to do it from another computer and it doesn't work either.

## To Reproduce (required)
Just try to compile the custom actions!!!

This is the function that was working and it no longer wants to compile the platform

import 'package:uuid/uuid.dart';

Future<String> uuidv1() async {
  var uuid = Uuid();
  var v1 = uuid.v1();

  return v1;
  // Add your function code here!
}

### Bug Report Code (required, generate from Widget Tree)
Unknown error compiling custom code. A common cause is a custom widget or action whose name in the code does not match the name provided in the editor.

## Context (required)
compilation for custom actions

## Screenshots / recordings
[]
https://downloads.intercomcdn.com/i/o/668124256/93a371ef4a0a6d97fa99ea0c/image.png?expires=1675863506&signature=5587489e2a95fb0ae7dd061e05579659b4d7bd4f82336687c2ef317048fbe857
https://downloads.intercomcdn.com/i/o/668124419/55f714916b7b2f9521395527/image.png?expires=1675863506&signature=1e6b5c762f083771e5fbad11b058edeed6ccf73f6c6f7d35e27f16aee9dbf3bf

## Your environment

* Version of FlutterFlow used: 3.0
* Platform (e.g. Web, MacOS Desktop): WEB but i try to solve compiling on a macOS and doesnt work too
* Browser name and version: chrome Versión 109.0.5414.120 (Build oficial) (64 bits)
* Operating system and version (desktop or mobile): windows 11 Versión	10.0.22000 compilación 22000

**Comment by guillo270120** [comment]
yes the code works but in my FF doesn´t compile

**Comment by guillo270120** [comment]
It was working fine but about 10 hours ago the custom actions no longer compiled for me

**Comment by hariprasadms** [comment]
Did you try coping to new one and compile. If not yet please try this and confirm. Thanks

**Comment by guillo270120** [comment]
I found this report and it's the same thing that happens to me, only it doesn't throw me errors, the function just doesn't compile

https://github.com/FlutterFlow/flutterflow-issues/issues/443

**Comment by guillo270120** [comment]
after duplicating the project, it does not compile neither on the original nor on the copy

**Comment by hariprasadms** [comment]
Ya thanks for confirming. It is an issue. I think there is a fix already made and it should be available in next updates.

**Comment by guillo270120** [comment]
of course and I know that they will solve it but while I cannot deliver my project to the client, I would greatly appreciate your support as soon as possible

**Comment by guillo270120** [comment]
Hello, will you have a tentative date or time for the deployment of the solution to this error?

**Comment by guillo270120** [comment]
Notwithstanding not being able to compile, now after the update I cannot work with the downloaded code since it is giving an error in the flutterflow libraries

**Comment by hariprasadms** [comment]
Hi @guillo270120, Can you please share bug report code of your project?

**Comment by younicornsJH** [comment]
I can confirm same problem here - basically compiling downloaded code using `$ flutter packages pub run build_runner build --delete-conflicting-outputs` and `flutter build web` yields hundreds of errors
Was working flawlessly last week
It seems you removed some needed libraries from /lib/flutter_flow/ folder: 

```
lib/flutter_flow/flutter_flow_util.dart:20:1:
Error: Error when reading 'lib/flutter_flow/uploaded_file.dart': Error reading 'lib/flutter_flow/uploaded_file.dart'  (No such file or directory)
```

**Comment by hariprasadms** [comment]
Hi, Please share your project link. We can have a quick look now.

**Comment by guillo270120** [comment]
could i send privatelly?

**Comment by guillo270120** [comment]
as a customers we need solutions from FF please

**Comment by aemelyanovff** [comment]
> I can confirm same problem here - basically compiling downloaded code using `$ flutter packages pub run build_runner build --delete-conflicting-outputs` and `flutter build web` yields hundreds of errors Was working flawlessly last week It seems you removed some needed libraries from /lib/flutter_flow/ folder:
> 
> ```
> lib/flutter_flow/flutter_flow_util.dart:20:1:
> Error: Error when reading 'lib/flutter_flow/uploaded_file.dart': Error reading 'lib/flutter_flow/uploaded_file.dart'  (No such file or directory)
> ```

This is now fixed.

**Comment by hariprasadms** [comment]
Thanks @aemelyanovff - @aemelyanovff @guillo270120 - Please check your projects now and let us know how it goes. 
@guillo270120 - You can generate bug report code and share that same here. This code is secure so no need to worry.  

You can find bug report code here.

![image](https://user-images.githubusercontent.com/17435678/217924456-16426de3-34d9-4ae6-af24-91800b3cb8e2.png)

**Comment by guillo270120** [comment]
the second issue is solved but the first still in my prject this is my bug report IT4el8j6yIt1j7xa+KX5bcJB/TssKUQgULhEiuxEcAwbF5CqE5oufuj+QHZVOMO/dFY3OlmI8Tse/srSv9nhV+4HPiiCGos+yqhcFg7iIT+RSqrSEqW8YHwmHZtYGW3LlePx+VcFfJIrDwREnhvSXw==

**Comment by hariprasadms** [comment]
HI, Thanks share for this details. I will get back to you with a fix. No worries. Thanks

**Comment by hariprasadms** [comment]
Hi @guillo270120 - The custom code error should be fixed now. Please check. If it is working, please close the issue. Thanks

**Comment by guillo270120** [comment]
Thx for the support !!!

**Comment by Tomaicomm** [comment]
It looks like I faced the same issue.
Someone above mentioned that it occured after duplication of the project.
It did not happened before - but this time it happens.
Before it 2 things happened.
1. I started to use a TEAM code for functions and actions - all was in tact
2. We recently duplicated the project - and the error popped up around this time.
Can anyone help?

<img width="385" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/128980409/f471bbc0-4d0d-49c1-863b-62e56e55162f">
